app/assets/stylesheets/_navbar.scss

```diff
@@ -1,66 +1,182 @@
 .navbar {
+  padding: 0 10px;
   margin-bottom: 0;
-  min-height: 0;
-  font-size: 12px;
-  background: $lightgrey-2;
-  border: 0;
   border-radius: 0;
 
-  .nav-pills {
-    .nav-item {
-      &.active {
-        .nav-link {
-          background-color: transparent;
-          border-bottom: 2px solid $darkred;
-          color: $darkred;
-        }
+  &-logo {
+    padding: 10px 15px;
+  }
+
+  &-nav > li > a {
```

Comment:
if I write on another way, I wouldn't be able to override Bootstrap's own selector :/ it has priority being written like this